src/content/docs/en/pages/build-journey/edit-edge-app/edit-cache-settings/cache-settings.mdx

```diff
@@ -53,15 +53,15 @@ The maximum and default TTL values for the edge should be between `60` and `3153
 A TTL zero isn't the same as a cache bypass. For more details, read about the difference between these settings on the [Bypass Cache behavior](/en/documentation/products/build/edge-application/rules-engine/#bypass-cache) documentation.
 :::
 
-### Slice files
+### Large File Optimization
 
-Instead of downloading a large content file and risking timeouts or connection terminations, [files can be sliced](/en/documentation/products/edge-application/cache-settings/#slice-settings) in pieces and cached on-demand.
+Instead of downloading a large content file and risking timeouts or connection terminations, [files can be segmented](/en/documentation/products/edge-application/cache-settings/#large-file-optimization) in pieces and cached on-demand.
```

Comment:
```suggestion
Instead of downloading a large content file and risking timeouts or connection terminations, [files can be fragmented](/en/documentation/products/edge-application/cache-settings/#large-file-optimization) in pieces and cached on-demand.
```